src/uwtools/scheduler/slurm.py

```diff
@@ -0,0 +1,127 @@
+# from io import StringIO
+from .scheduler import Scheduler
+
+__all__ = ['Slurm']
+
+
+class Slurm(Scheduler):
+    """
+    Class to construct Slurm job cards.
+    For SBATCH reference see: https://slurm.schedmd.com/sbatch.html
+    """
+
+    _DIRECTIVE = '#SBATCH'
+    _MAPPING = {'account': '--account',
+                'partition': '--partition',
+                'queue': '--qos',
+                'jobname': '--job-name',
+                'stdout': '--output',
+                'stderr': '--error',
+                'walltime': '--time',
+                'env': '--export',
+                'nodes': '--nodes',
+                'tasks': '--ntasks',
+                'tasks_per_core': '--ntasks_per-core',
+                'tasks_per_node': '--ntasks_per-node',
+                'memory': '--mem',
+                'exclusive': '--exclusive',
+                'debug': '--verbose'}
+
+    def __init__(self, config: dict, *args, **kwargs):
+        """
+        Constructor for Slurm scheduler.
+
+        Parameters
+        ----------
+        config : dict
+        """
+
+        super().__init__(config, *args, **kwargs)
+
+        self.get_accounting
+        self.get_resources
+        self.get_native
+        self.get_env
+
+    @property
+    def get_accounting(self):
+        """
+        Generate the accounting specific items of the job card.
+        E.g. jobname, queuing, partitions, accounts etc.
+        """
+        strings = []
+        if 'jobname' in self.specs:
```

Comment:
Yes, I saw the comment and I appreciate it.  The duplication should be reduced, certainly.
I chose this route because I realized that `slurm` and `PBS` don't follow the same association.  And when `LSF` comes through, it will be even more so.  
Hence, in order to keep them self-contained, I chose to implement the concrete methods in their respective classes.
I am totally open to revisit this.